src/utils/openCageApiUtils.ts

```diff
@@ -27,7 +27,10 @@ export class OpenCageApiUtils {
 	public async addressLookup(address: string): Promise<Address | undefined> {
 		return opencage
 			.geocode({ q: address, key: this.apiKey })
-			.then((data) => this.reformatGeocodeResult(data?.results?.[0]))
+			.then((data) => {
+				data?.results?.map((x) => console.log(x.formatted));
```

Comment:
This creates seperate `console.log` statements. If you want to send this as debug info, you should make it as single log statement.